src/display/text_layer.js

```diff
@@ -456,6 +469,25 @@ class TextLayer {
     return canvasContext;
   }
 
+  /**
+   * @returns {number} The minimum font size enforced by the browser
```

Comment:
This method no longer returns anything.
```suggestion
   * Compute the minimum font size enforced by the browser.
```